core/order_test.go

```diff
@@ -356,3 +358,27 @@ func TestOpenBazaarNode_CalculateOrderTotal(t *testing.T) {
 		t.Error("Calculated wrong order total")
 	}
 }
+
+func TestOpenBazaarNode_GetOrder(t *testing.T) {
+	node, err := test.NewNode()
+	if err != nil {
+		t.Error(err)
```

Comment:
When a test dependency fails, we should abort the rest of the test run (as we already know it can't pass, if the dep fails to be produced). Would prefer `t.Fatal(err)` instead which would ensure the test is exited/returned after the error state is reported.

Reply:
done